Host a Static Website with Hugo and NGINX.md

```diff
@@ -451,16 +450,16 @@ It is very possible that you may attempt to stop or reload the Nginx server and
     Failed to stop nginx.service: The name org.freedesktop.PolicyKit1 was not provided by any .service files
     See system logs and 'systemctl status nginx.service' for details.
 
-To start, stop, or reload Nginx it is necessary to use the `sudo` prefix along with the command to stop, reload or start Nginx. That is, `sudo systemctl stop nginx`, `sudo systemctl reload nginx`, `sudo systemctl start nginx`.
+To start, stop, or reload Nginx it is necessary to prefix the command with `sudo`. For example: `sudo systemctl reload nginx`.
 
 ### Hugo Server did not Shutdown Using **CTRL+C**
 
-There are times when shutting down the server with **CTRL+C** doesn't do the trick. This may be apparent the next time we attempt to start-up the Hugo server. For example, the Terminal console may display something to the effect, "Web Server is available at `http://localhost:12345`". 
+There are times when shutting down the server with **CTRL+C** doesn't do the trick. This may be apparent the next time we attempt to start-up the Hugo server. For example, the Terminal console may display something similar to `web server is available at http://localhost:12345`. 
 
-Here is the scenario that can result in this condition:
+Here is a scenario that can result in this condition:
```

Comment:
So rarely do we want to say this is the **only** way to do something. Leave the possibilities open. It is "a" way.

Reply:
Agree.